apiDocs/wrhs-spec.md

```diff
@@ -28,9 +28,7 @@ Gets the file assets for a given package-environment-version
 | 404 | Not Found |
 | 500 | Internal Server Error |
 
-##### Security
-
-unauthed
+null
```

Comment:
the unauthed-> null change is because the library we use outputs null when the security is left empty in the jsdocs. Previously, we just manually changed the generated output to use `unauthed` instead of `null`